bokehjs/examples/anscombe/.gitignore

```diff
@@ -1 +1,2 @@
 anscombe.js
+anscombe-*.png
```

Comment:
Can't we just add a single `.gitignore` one level up with `*.js` an `*.png` ? Having those definitely seems like the exception. If we need to add a `.js` we can make git do that on a case by case basis, instead of having to remember to add a custom `.gitignore` to every new example.